cypress/e2e/create_event_spec.js

```diff
@@ -16,7 +16,9 @@ describe('Create event', () => {
   const uploadHeader = () => {
     // Upload file
     cy.upload_file(
-      c('ImageUploadField__coverImage') + ' ' + c('UploadImage__dropArea'),
+      c('ImageUploadField__coverImage') +
+        ' ' +
+        c('UploadImage__placeholderTitle'),
```

Comment:
Thanks to @eikhr for spotting that Cypress was failing because it was trying to click in the center of the image drop area, where the image icon was located, and that simply making it click on the text below instead solved the issue. Strange behavior by Cypress, but it's because the icon is now a div instead of being text (a font to be precise).